modules/local/metabinner_metabinner/main.nf

```diff
@@ -0,0 +1,52 @@
+process METABINNER_METABINNER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metabinner:1.4.4--hdfd78af_0' :
+        'quay.io/biocontainers/metabinner:1.4.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta), path(kmer), path(depth)
+
+    output:
+    tuple val(meta), path("*.tsv.gz") , emit: membership
+    tuple val(meta), path("*.log.gz") , emit: log
+    path "versions.yml"               , emit: versions
+
+    script:
+    def args   = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def min_contig_size = task.ext.min_contig_size ?: "1000"
+    def VERSION = '1.4.4-0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    # unzip kmer file
+    zcat $kmer > kmer_profile.csv
+
+    # create coverage profile in Metabinner format
+    zcat ${depth} | awk '{if (\$2>${min_contig_size}) print \$0 }' | cut -f -1,4- > coverage_profile.tsv
```

Comment:
I think this is fine for now, but if we consider in future to put this module in the modules repo, we should move this elsewhere to make the module more reusable